staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -281,6 +287,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	ServerSideFieldValidation: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.29
 
+	SkipUnauthenticatedHTTP2DOSMitigation: {Default: false, PreRelease: featuregate.Deprecated}, // remove in 1.30
```

Comment:
Given the scenarios raised in review (loadbalancer that already mitigates the issue, etc), I'm not yet confident we'll be able to just assume this and drop the gate in 1.30. I'd drop or change that comment.